sw/device/tests/sensor_ctrl_alerts.c

```diff
@@ -100,6 +100,10 @@ static void check_alert_state(dif_toggle_t fatal) {
  *  Finally, check for correct capture of cause in alert handler.
  */
 static void test_event(uint32_t idx, dif_toggle_t fatal) {
+  // Enable the alert on the sensor_ctrl side
```

Comment:
agreed, it would be good to add such a negative test.

we can do that as part of the V signoff for this block (the block is only tested at the top-level anyway).